apps/src/javalab/javalab-settings.module.scss

```diff
@@ -0,0 +1,18 @@
+@use 'control-buttons.module';
```

Comment:
If I understand `@use` correctly, it seems like I'm basically prepending the entire `control-buttons` file here. I think you can be more specific if you're trying to use specific variables or mixins, but not if you're just trying to get access to a class (I specifically use the `buttonWhite` class in `JavalabSettings` that is generated in `control-buttons`)

Reply:
That's my understanding as well